src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -83,7 +88,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         private CSharpParseOptions(CSharpParseOptions other) : this(
-            languageVersion: other.LanguageVersion,
+            languageVersion: other.SpecifiedLanguageVersion,
```

Comment:
That would be an unjustified micro-optimization of code that isn't on a hot path, so it is exactly the kind of thing we should avoid. [http://c2.com/cgi/wiki?PrematureOptimization](http://c2.com/cgi/wiki?PrematureOptimization)

---
In reply to: [74520871](https://github.com/dotnet/roslyn/pull/12969#discussion_r74520871) [](ancestors = 74520871)